extensions/oidc-client/runtime/src/main/java/io/quarkus/oidc/client/runtime/OidcClientConfig.java

```diff
@@ -172,6 +172,14 @@ public String getGrantType() {
      */
     Map<String, String> headers();
 
+    /**
+     * Token refresh interval. By default, Quarkus refreshes tokens that are expired tokens or within access
```

Comment:
@michalvavrik Can you please align it with the docs content, perhaps something like

```
Token refresh interval.

By default, OIDC client refreshes the token during the current request, when it detects that it has expired, or nearly expired if the {@link #refresh_token_time_skew} is configured. But, when this property is configured, OIDC client can refresh the token asynchronously in the configured interval.

This property is only effective with OIDC client filters and other {@link AbstractTokensProducer} extensions but not when you use  {@link OidcClient#getTokens} API directly.
```

Perhaps something like that ? Please tune it as preferred.